### ISSUE TITLE
Remove realpath?

### ISSUE THREAD

**Issue by johnstonphilip** [wp-plugin-sidekick/wpps-scripts#13]
I wonder if we can remove the need to use realpath? It seems it's not a default on some systems. 
https://github.com/wp-plugin-sidekick/wpps-scripts/blob/b733ad6d32d3f0315da1abe490ecb1d55f0c7e6e/setup.sh#L18

Related:
https://github.com/ohmyzsh/ohmyzsh/pull/4106/files

Thoughts @marksabbath, @kienstra? Is that another way we could do this same thing without needing realpath?

**Comment by kienstra** [comment]
Yeah, removing `realpath` sounds good. I'll look at ways we might do that.
